models/public/mixnet-l/README.md

```diff
@@ -0,0 +1,73 @@
+# mixnet-l
+
+## Use Case and High-Level Description
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. There are three MixNet architectures -
+`MixNet-S` (Small), `MixNet-M` (Middle), `MixNet-L` (Large). The main differences are using
+MixConv with different kernel sizes and number of layers. Using `MixNet-L` allows to achieve greater accuracy.
+Use this [link](https://arxiv.org/abs/1907.09595) to learn more about MixNet architectures.
```

Comment:
```suggestion
Refer to original [article](https://arxiv.org/abs/1907.09595) to learn more about MixNet architectures.
```

Reply:
Please see new version of `Use Case and High-Level Description`. Perhaps it is a better variant (suggested Aidova Ekaterina).

Reply:
ok with small correction. Seems ImageNet dataset looks more appropriate than ImageNet image database